datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -2597,6 +2597,7 @@ SELECT
 # test_source_sorted_builtin
 query TT
 EXPLAIN SELECT
+  ts,
```

Comment:
this change is to demonstrate that the `ts` column is not unique in the first `5` values and thus that a different output order is acceptable